Resources/Private/JavaScript/media-module/src/components/TopBar/TopBar.tsx

```diff
@@ -24,7 +25,10 @@ const TopBar: React.FC = () => {
     const { selectionMode } = useMediaUi();
     const classes = useStyles({ selectionMode });
 
-    const components = useMemo(() => [ClipboardActions, SearchBox, TypeFilter, ViewModeSelector], []);
+    const components = useMemo(
+        () => [ClipboardActions, SearchBox, TypeFilter, SortOrderSelector, ViewModeSelector],
```

Comment:
Hopefully the positioning is OK here, I wasn't sure, if there are any designs / mockups on what goes where

Reply:
Yes that's fine, there are no official guidelines yet. I mostly used the ideas collected in the discuss topics about the media management.